src/evaluator.rs

```diff
@@ -303,6 +303,7 @@ impl<'src, 'run> Evaluator<'src, 'run> {
 }
 
 #[cfg(test)]
+#[allow(dead_code)]
```

Comment:
Did something change which makes this necessary?

Reply:
Probably a leftover of some testing. Will remove